x-pack/legacy/plugins/siem/public/pages/case/components/shared_imports.ts

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
nit -> maybe we can move out the shared_omports from detection to a global folder so bot of them can re-use the same